CHANGELOG.md

```diff
@@ -9,6 +9,9 @@
 * C#
   * Add support for the String List metric type.
   * Enable generating the C# APIs using the glean_parser.
+* Python
```

Comment:
Can you also add a "General" changelog entry for this PR?

Reply:
Sure. I was not sure that was necessary since I am not introducing any new functionality.